app/assets/javascripts/bastion/components/bst-container-scroll.directive.js

```diff
@@ -34,8 +34,14 @@ angular.module('Bastion.components').directive('bstContainerScroll', ['$window',
                         offset = offset + bottomPadding;
                     }
 
-                    element.outerHeight(windowHeight - offset);
-                    element.height(windowHeight - offset);
+                    newElementHeight = windowHeight - offset;
+
+                    if (newElementHeight <= 50) {
+                        newElementHeight = 200;
```

Comment:
Anything specific that you chose these values? I played around a bit and found 100/300 to be a bit nicer to me.